### DISCUSSION TITLE
connect Pi to a PC for easier updates

### DISCUSSION THREAD

**Discussion by Piattica** [probonopd/MiniDexed#578] [Q&A]
To update performances and working on performance.ini of Minidexed SD card
I continously remove and insert SD card in Raspberry.
Is there another way to connect Raspberry and PC to allow me to work from PC on the SD card inserted in PI ?

**Comment by probonopd** [comment]
For voices, you can use an editor on the PC that can send voice data as MIDI sysex messages to MiniDexed.

However, for performanes we don't have something like this yet.

My idea would be to implement 2 new custom MIDI sysex commands:
* Write to file (parameters: path to file, binary data that shall be written to the file)
* Delete file (parameter: path to file)

Tools like https://github.com/BobanSpasic/MiniDexedLibrarian could then possibly use that.
